### PR TITLE
Enhancement(Feeder) Change TAGS from string to string array (#961)

### LINKED ISSUE #961: Karmor Log(Alert), TAG field changes from string to string array
## Feature Request

In policies there are multiple tags in a single array, due to a single policy can be linked to different use cases, changing it to a string array will send the partitioned Tags in output, which will be easy to handle in different SIEM integration and further representation.

Currently, in the output, the Tags are coming in this format combined as a single string 

```
Tags: MITRE, MITRE_T1552_unsecured_credentials
```
Changing the output to the below output will automatically portion the tags.
```

"Tags": [
            "MITRE",
            "MITRE_T1552_unsecured_credentials"
        ],

```

- [ ] Create new protobuf entry
- [ ] Add a task to deprecate "tags" field in v0.10.

### PR DESCRIPTION
**Pull request by vishnusomank**
**Purpose of PR?**:

To show/send an array of Tags to output, making it easy to handle in different SIEM integration and other representation.

**Updates in:**

- feeder/feeder.go to include new tags field `ATags`
- protobuf to inlcude `ATags` array
- feeder/policyMatcher.go to write tags array to new field `ATags`
- types/types.go to include `ATags` array in the Alert structure

Fixes [#961](https://github.com/kubearmor/KubeArmor/issues/961)

**Does this PR introduce a breaking change?** NO

**Checklist:**
- [x] Bug fix. Fixes [#961](https://github.com/kubearmor/KubeArmor/issues/961)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

Signed-off-by: vishnusomank <vishnu@accuknox.com>

**Telemetry data:**
```
➜  ~ cat /tmp/kubearmor.log | grep wordpress | grep -i "permission denied" | jq .
{
  "timestamp": 1667542698,
  "updatedTime": "2022-11-04T06:18:18.230978Z",
  "hostName": "zero-knox",
  "namespaceName": "wordpress-mysql",
  "podName": "wordpress-765f89bf9f-mglrn",
  "labels": "app=wordpress",
  "containerID": "a2eaef4651d1917ee9be3e54b1d3c2639ae4582ce344df46c811d970696675f6",
  "containerName": "wordpress",
  "containerImage": "wordpress:4.8-apache@sha256:6216f64ab88fc51d311e38c7f69ca3f9aaba621492b4f1fa93ddf63093768845",
  "hostPPid": 83228,
  "hostPid": 83469,
  "ppid": 201,
  "pid": 208,
  "uid": 0,
  "parentProcessName": "/bin/bash",
  "processName": "/usr/bin/apt",
  "enforcer": "AppArmor",
  "policyName": "wordpress-wordpress-4-8-apache-least-functionality-execute-package-management-process-in-container",
  "severity": "5",
  "tags": "SI-4,NIST,NIST_800-53_CM-7(4),NIST_800-53_SI-4,process",
  "atags": [
    "SI-4",
    "NIST",
    "NIST_800-53_CM-7(4)",
    "NIST_800-53_SI-4",
    "process"
  ],
  "message": "Alert! Execution of package management process inside container is denied",
  "type": "MatchedPolicy",
  "source": "/bin/bash",
  "operation": "Process",
  "resource": "/usr/bin/apt",
  "data": "syscall=SYS_EXECVE",
  "action": "Block",
  "result": "Permission denied"
}

```

**Screenshots:**

image

### CODE REVIEW

File: KubeArmor/types/types.go [hunk 1]
```diff
@@ -211,9 +211,10 @@ type Log struct {
 	PolicyName string `json:"policyName,omitempty"`
 
 	// severity, tags, message
-	Severity string `json:"severity,omitempty"`
-	Tags     string `json:"tags,omitempty"`
-	Message  string `json:"message,omitempty"`
+	Severity string   `json:"severity,omitempty"`
+	Tags     string   `json:"tags,omitempty"`
```
Comment (rajaSahil) [inline KubeArmor/types/types.go:215]: Do we need tags? We do have array of tags, can't we use it in place of tags?

Comment (vishnusomank) [reply KubeArmor/types/types.go:215]: @rajaSahil  yes, we need tags. Please refer to this conversation https://github.com/kubearmor/KubeArmor/issues/961#issuecomment-1301616238

Comment (rajaSahil) [reply KubeArmor/types/types.go:215]: Oh okay.

**Review by nyrahul — APPROVED** [review]
LGTM 👍

**Pull request merged**